src/diffcalc_API/server.py

```diff
@@ -46,6 +47,8 @@ async def server_exceptions_middleware(request: Request, call_next):
         return await call_next(request)
     except Exception as e:
         # you probably want some kind of logging here
```

Comment:
This should either be implemented or made into a proper `TODO`

Reply:
Good idea - this will be a future PR maybe. I need to actually have some logging involved.